src/main/java/io/stargate/sgv2/jsonapi/config/OperationsConfig.java

```diff
@@ -109,4 +113,59 @@ interface LwtConfig {
     @WithDefault("3")
     int retries();
   }
+
+  /** Cassandra/AstraDB related configurations. */
+  @NotNull
+  @Valid
+  DatabaseConfig databaseConfig();
+
+  interface DatabaseConfig {
+
+    /** Database type can be <code>cassandra</code> or <code>astra</code>. */
+    @WithDefault(CASSANDRA)
+    String type();
+
+    /** Username when connecting to cassandra database (when type is <code>cassandra</code>) */
+    @Nullable
+    @WithDefault("cassandra")
+    String userName(); // TODO move to request info
+
+    /** Password when connecting to cassandra database (when type is <code>cassandra</code>) */
+    @Nullable
+    @WithDefault("cassandra")
+    String password(); // TODO move to request info
+
+    /** Fixed Token used for Integration Test authentication */
+    @Nullable
+    @WithDefault("not in tests")
+    String fixedToken();
+
+    /** Cassandra contact points (when type is <code>cassandra</code>) */
+    @Nullable
+    @WithDefault("127.0.0.1")
+    List<String> cassandraEndPoints();
+
+    /** Cassandra contact points (when type is <code>cassandra</code>) */
+    @Nullable
+    @WithDefault("9042")
+    int cassandraPort();
+
+    /** Secure connect bundle path (when type is <code>astra</code>) */
+    @Nullable
```

Comment:
Do we need this?

Reply:
Removed in this commit 63e3a1e7e8277a98ef6e117d10b02272cef0d92d